src/transformers/models/whisper/configuration_whisper.py

```diff
@@ -129,8 +129,6 @@ class WhisperConfig(PretrainedConfig):
             Begin of stream token id.
         eos_token_id (`int`, *optional*, defaults to 50257):
             End of stream token id.
-        tie_word_embeddings (`bool`, *optional*, defaults to `True`):
```

Comment:
The default value in `PretrainedConfig` is `True`. We don't need to specify here. Removing here just for housekeeping. cc @ArthurZucker